org.gnome.Weather.json

```diff
@@ -59,8 +59,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gnome-weather/40/gnome-weather-40.0.tar.xz",
-                    "sha256": "2a35a73ab2408762181d8650b037205c17ef7bcb8dff3cf0b34af1a2de66aeef"
+                    "url": "https://download.gnome.org/sources/gnome-weather/40/gnome-weather-40.1.tar.xz",
```

Comment:
Actually, the tarball hasn't been uploaded yet.

Reply:
Yeah, there are [some issues](https://gitlab.gnome.org/GNOME/gnome-weather/-/merge_requests/85#note_1246182) with the upload.

Reply:
The tarball has been uploaded today.

Reply:
The tarball has been uploaded today.

Reply:
We'll need to update the hash of the tarball first. I will try to do so later today.

Reply:
Done.